app/models/adoptions_request.rb

```diff
@@ -0,0 +1,5 @@
+class AdoptionsRequest < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :gems_adoption
+  validates :gems_adoption_id, :user_id, presence: true
```

Comment:
I think it's a better idea to validate the presence of the association than only the presence of the foreign key and validations support that. Do you agree?
```ruby
validates :gems_adoption, :user, presence: true
```